styles.js

```diff
@@ -1,13 +1,77 @@
 import { createGlobalStyle } from "styled-components";
 
-export default createGlobalStyle`
+export const accentColors = ["#a9ead8", "#ade7e2", "#b0e2eb", "#b1d5ec", "#b0c9eb"];
+
+const GlobalStyles = createGlobalStyle`
+  :root {
+    --white: #ffffff;
+    --darktext: #27273a;
+    --background: #f5f7fd;
+    --gray-light: #ebebf3;
+
+    --boxshadow-primary: hsl(0 0% 0% / 15%) 0px 2px 12px 0px;
+    --boxshadow-secondary: hsl(0 0% 0% / 25%) 0px 7px 6px -5px;
+
+    --fontfamily-regular: 'Inter', sans-serif;
+    --fontfamily-special: 'Cutive', monospace;
```

Comment:
With next.js 13 comes a really cool font-optimisation (https://nextjs.org/docs/pages/building-your-application/optimizing/fonts). Please have a look at it. I applied it to our project (commit 'next-font-optimisation') and just commented out what is not needed anymore to give you an example.

Reply:
This looks like a very useful new feature. Thanks for showing me. 😃 
The only downside I can see right now is that it's defined in _app.js. I would always look for anything related to fonts in a styles.js or similar. I guess you just have to get used to it.

Reply:
That's right. Maybe there is a workaround to use it in styles.js